binding/vue/src/porcupine.ts

```diff
@@ -12,37 +12,34 @@
 import { WebVoiceProcessor } from '@picovoice/web-voice-processor';
 
 import {
+  BuiltInKeyword,
+  PorcupineDetection,
   PorcupineKeyword,
+  PorcupineModel,
+  PorcupineOptions,
   PorcupineWorker,
-  PorcupineWorkerFactory
-} from "@picovoice/porcupine-web-core";
-
-/**
-* Type alias for PorcupineWorkerFactory arguments.
-*/
-export type PorcupineWorkerFactoryArgs = {
-  accessKey: string;
-  keywords: Array<PorcupineKeyword | string> | PorcupineKeyword | string;
-  start?: boolean;
-};
+} from '@picovoice/porcupine-web';
 
 /**
  * Type alias for Porcupine Vue Mixin.
  * Use with `Vue as VueConstructor extends {$porcupine: PorcupineVue}` to get types in typescript.
  */
 export interface PorcupineVue {
-  $_ppnWorker_: Worker | null;
-  $_webVp_: WebVoiceProcessor | null;
+  $_porcupine_: PorcupineWorker | null,
   init: (
-    porcupineFactoryArgs: PorcupineWorkerFactoryArgs,
-    porcupineFactory: PorcupineWorkerFactory,
-    keywordCallback: (label: string) => void,
-    readyCallback: () => void,
-    errorCallback: (error: string | Error) => void) => void;
-  start: () => Promise<boolean>;
-  stop: () => Promise<boolean>;
-  pause: () => boolean;
-  delete: () => void;
+    accessKey: string,
+    keywords: Array<PorcupineKeyword | BuiltInKeyword> | PorcupineKeyword | BuiltInKeyword,
+    keywordDetectionCallback: (porcupineDetection: PorcupineDetection) => void,
+    model: PorcupineModel,
+    options?: PorcupineOptions,
+  ) => Promise<void>,
+  start: () => Promise<void>,
+  pause: () => Promise<void>,
```

Comment:
Just need to remove `pause` and add `release` here to match the interface we settled on